### ISSUE TITLE
bbox loss always 0 during training, and acc 100%, but no detection results.

### ISSUE THREAD

**Issue by GlennLoveZhen0124** [open-mmlab/mmdetection#4355]
/root/mmdetection/mmdet/datasets/custom.py:154: UserWarning: CustomDataset does not support filtering empty gt images.
  'CustomDataset does not support filtering empty gt images.')
2020-12-26 22:09:25,810 - mmdet - INFO - load model from: torchvision://resnet50
2020-12-26 22:09:26,112 - mmdet - WARNING - The model and loaded state dict do not match exactly

unexpected key in source state_dict: fc.weight, fc.bias

2020-12-26 22:09:32,945 - mmdet - INFO - Start running, host: root@ligaoyang_6, work_dir: /root/mmdetection/work_dirs
2020-12-26 22:09:32,945 - mmdet - INFO - workflow: [('train', 1)], max: 12 epochs
2020-12-26 22:10:08,314 - mmdet - INFO - Epoch [1][50/527]	lr: 2.500e-03, eta: 1:13:37, time: 0.704, data_time: 0.049, memory: 3787, loss_rpn_cls: 0.1513, loss_rpn_bbox: 0.0000, loss_cls: 0.2045, acc: 98.5449, loss_bbox: 0.0000, loss: 0.3559
2020-12-26 22:10:41,882 - mmdet - INFO - Epoch [1][100/527]	lr: 2.500e-03, eta: 1:11:18, time: 0.671, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.1802, acc: 100.0000, loss_bbox: 0.0000, loss: 0.1802
2020-12-26 22:11:15,701 - mmdet - INFO - Epoch [1][150/527]	lr: 2.500e-03, eta: 1:10:21, time: 0.676, data_time: 0.009, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0905, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0905
2020-12-26 22:11:49,085 - mmdet - INFO - Epoch [1][200/527]	lr: 2.500e-03, eta: 1:09:22, time: 0.667, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0497, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0497
2020-12-26 22:12:22,456 - mmdet - INFO - Epoch [1][250/527]	lr: 2.500e-03, eta: 1:08:33, time: 0.668, data_time: 0.009, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0307, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0307
2020-12-26 22:12:55,585 - mmdet - INFO - Epoch [1][300/527]	lr: 2.500e-03, eta: 1:07:45, time: 0.663, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0214, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0214
2020-12-26 22:13:29,316 - mmdet - INFO - Epoch [1][350/527]	lr: 2.500e-03, eta: 1:07:12, time: 0.675, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0160, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0160
2020-12-26 22:14:03,046 - mmdet - INFO - Epoch [1][400/527]	lr: 2.500e-03, eta: 1:06:37, time: 0.674, data_time: 0.007, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0124, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0124
2020-12-26 22:14:36,250 - mmdet - INFO - Epoch [1][450/527]	lr: 2.500e-03, eta: 1:05:56, time: 0.664, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0098, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0098
2020-12-26 22:15:10,211 - mmdet - INFO - Epoch [1][500/527]	lr: 2.500e-03, eta: 1:05:26, time: 0.679, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0082, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0082
2020-12-26 22:15:28,147 - mmdet - INFO - Saving checkpoint at 1 epochs
2020-12-26 22:16:05,196 - mmdet - INFO - Epoch [2][50/527]	lr: 2.500e-03, eta: 1:01:59, time: 0.727, data_time: 0.049, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0062, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0062
2020-12-26 22:16:38,036 - mmdet - INFO - Epoch [2][100/527]	lr: 2.500e-03, eta: 1:01:31, time: 0.657, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0055, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0055
2020-12-26 22:17:11,769 - mmdet - INFO - Epoch [2][150/527]	lr: 2.500e-03, eta: 1:01:10, time: 0.675, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0048, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0048
2020-12-26 22:17:45,323 - mmdet - INFO - Epoch [2][200/527]	lr: 2.500e-03, eta: 1:00:45, time: 0.671, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0043, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0043
2020-12-26 22:18:18,367 - mmdet - INFO - Epoch [2][250/527]	lr: 2.500e-03, eta: 1:00:16, time: 0.661, data_time: 0.009, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0039, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0039
2020-12-26 22:18:52,190 - mmdet - INFO - Epoch [2][300/527]	lr: 2.500e-03, eta: 0:59:52, time: 0.676, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0034, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0034
2020-12-26 22:19:25,987 - mmdet - INFO - Epoch [2][350/527]	lr: 2.500e-03, eta: 0:59:26, time: 0.675, data_time: 0.008, memory: 3787, loss_rpn_cls: 0.0000, loss_rpn_bbox: 0.0000, loss_cls: 0.0031, acc: 100.0000, loss_bbox: 0.0000, loss: 0.0031

my train goes wierd, the bbox loss is 0 from the beginning, acc becomes 100% from the second batch. And the detection result in inference is empty! Please help

I use faster_rcnn_r50_fpn_1x_coco.py as config file

**Comment by ZwwWayne** [comment]
Please use the issue template of `reimplementation questions`. We are not able to do any help here with your current information.

**Comment by zwqnju** [comment]
I also encounter your issue when I use COCO format dataset. But when I create a new dataset like KittiTinyDataset in tutorial, instead of reorganizing my dataset into COCO format,  everything seems OK.

**Comment by waddington-ou-phd-1** [comment]
Did you find a solution to this? I am having the same problem.

**Comment by rain-1227** [comment]
Did you find a solution to this? I am having the same problem

**Comment by ZwwWayne** [comment]
Since no more information is provided, this issue will be closed. Feel free to create a new issue using the `error template` to report your bug.
